.changeset/cuddly-pets-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": major
```

Comment:
Did you mean a major change to Wrangler here? I feel like this should be considered a patch change given it is just an internal refactoring